### DISCUSSION TITLE
How to remove my curent save?

### DISCUSSION THREAD

**Discussion by zaze06** [lxgr-linux/pokete#253] [Q&A] [answered]
Hi, I wonder if I can remove my current save? or create a new one and be able to select one like save slots?

**Comment by lxgr-linux** [comment, marked as answer]
The best way to restart your game would be to remove the `~/.local/share/pokete/pokete.json` file, which contains the save-states. Looking forward it might be a good idea to create a selection system when starting a game.

**Reply by zaze06** [reply]
Thanks :)
Also don't know what I did but my save recently disappeared so that might be something to look into, I can create an issue of it if you want too
